docs/general/run-a-node/set-up-your-node/run-validator.md

```diff
@@ -607,6 +607,17 @@ Note that in order to be elected in the validator set you need to have enough st
 
 :::
 
-## You're a Validator!
+## Oasis Metadata Registry
 
-If you've made it this far, you've properly connected your node to the network and you're now a validator on the Oasis Network.
+Congratulations, if you made it this far, you've properly connected your node 
```

Comment:
This paragraph appears out of place now. Can you move it before the Oasis Metadata Registry title?